flink-streaming-java/src/test/java/org/apache/flink/streaming/api/operators/collect/CollectSinkFunctionTest.java

```diff
@@ -136,9 +135,8 @@ void testConfiguredPortIsUsed() throws Exception {
                     .getConfiguration()
                     .setInteger(TaskManagerOptions.COLLECT_PORT, socket.getLocalPort());
             assertThatThrownBy(() -> functionWrapper.openFunction())
-                    .satisfies(
-                            FlinkAssertions.anyCauseMatches(
-                                    BindException.class, "Address already in use (Bind failed)"));
+                    .isInstanceOf(BindException.class)
+                    .hasMessageContaining("Address already in use");
```

Comment:
I wonder if there is any reason to go away from `FlinkAssertions.anyCauseMatches` ?
IIRC `FlinkAssertions.anyCauseMatches` also checks for containing message, so probably the only thing which should be changed is `"Address already in use (Bind failed)"` replace with `"Address already in use"`

Or did I miss anything?

Reply:
Thanks @snuyanzin for the review.

`FlinkAssertions.anyCauseMatches` will check all the causes, there will be additional overhead in abnormal conditions.
In this case, just check the outer exception is enough. 
I think it is unnecessary to use the `anycauSematches`method.

Reply:
Which one do you prefer?
I can make a change if we want to reduce the change.

Reply:
yes, makes sense, thanks for clarification